zulipterminal/config/keys.py

```diff
@@ -279,6 +279,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Show/hide stream information & modify settings',
         'key_category': 'stream_list',
     }),
+    ('TOPIC_DESC', {
+        'keys': ['i'],
+        'help_text': 'Show/hide topic information & modify settings',
+        'key_category': 'stream_list',
```

Comment:
How about ``topic_list``?

Reply:
Looking at similar popups/sections, we have one in the message-actions section (MSG_INFO in msg_actions), one in stream list actions section (STREAM_DESC in stream_list), but also user information in general (USER_INFO in general).

Based on that, it seems like it might be worth tidying the existing STREAM_DESC into an _INFO (so we can use a similar suffix in this commit). Until we restructure the help sections, to be consistent for now, we could either add the new hotkey to general (less tidy) or move it to a new topic_list section (and also move the existing user one into one two).

The latter option would possibly also allow adding an entry for ENTER in the user list for replying to a user, which we could possibly do in other cases too in some way, though we might want to hold off on that until we have more of a plan for that without adding too many options.